src/transformers/utils/import_utils.py

```diff
@@ -478,6 +478,8 @@ def is_torch_compile_available():
 
     import torch
 
+    # We don't do any version check here to support nighlies marked as 1.14. Ultimately needs to check version against
```

Comment:
FWIW, nightlies are now at 2.1.0.devXYZ

Reply:
Yes but old nightlies are in the wild, and some docker images are built with them (in particular the official Nvidia PyTorch docker image)